x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -37,6 +37,23 @@ async function deleteIndices(indices: string[], client: Client) {
   }
 }
 
+async function addUser(esClient: Client) {
+  try {
+    await esClient.transport.request({
+      method: 'POST',
+      path: '_security/user/endpoint_user',
+      body: {
+        password: 'changeme',
+        roles: ['superuser', 'kibana_system'],
+        full_name: 'endpoint user',
+      },
+    });
+    console.log('user endpoint_user added successfully!');
```

Comment:
Can we check if this user is already created before create it?

Reply:
addressed c21642a1703c592ef6e1dba51c771f4a41fe4541 and 0f37f21fe49ea2169d183d77a4565b3adf4eb56f

Reply:
okay so if you try to add an existing user the API doesn't add it again. This simplifies it more. ea9695d572f1bb180f1f9124652edb7b929d2bba